scripts/min_compatible_versions.py

```diff
@@ -13,8 +13,8 @@
 
 
 OUT_FILENAME = "min_compatible_versions.csv"
-OUT_DIRECTORIES = (".", "lib-injection")
-IGNORED_PACKAGES = {"setuptools"}
+OUT_DIRECTORIES = (".", "lib-injection/sources")
+IGNORED_PACKAGES = {"setuptools", "attrs", "pytest-randomly", "pillow", "botocore"}
```

Comment:
Probably a little more aggressive since I wasn't reading the file properly, I would be ok with you removing the other deps.